sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/JmsIOTest.java

```diff
@@ -74,7 +72,7 @@
 
   private BrokerService broker;
   private ConnectionFactory connectionFactory;
-  private ConnectionFactory connectionFactoryWithoutPrefetch;
+  private ConnectionFactory connectionFactoryWithSyncAcksAndWithoutPrefetch;
```

Comment:
Why not using this `ConnectionFactory` by default ? It doesn't have an impact on the other part of the IO.